e/etcd/etcd_ubi_8.5.sh

```diff
@@ -0,0 +1,56 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: etcd
+# Version	: v3.5.1
+# Source repo	: https://github.com/etcd-io/etcd
+# Tested on	: UBI 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Atharv Phadnis <Atharv.Phadnis@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=etcd
+PACKAGE_VERSION=v3.5.1
+PACKAGE_URL=https://github.com/etcd-io/etcd
+
+yum install -y git golang make
+
+HOME_DIR=`pwd`
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
```

Comment:
In my opinion, while dealing with go packages, try to clone and build under GOPATH.
```
export GOPATH=$HOME/go
mkdir -p $GOPATH/src/github.com/etcd-io/
cd $GOPATH/src/github.com/etcd-io/
git clone $PACKAGE_URL $PACKAGE_NAME
```
Please go through the GO documentation for `GOPATH` and `GOROOT`

Reply:
@kandarpamalipeddi Actually in case of this package, it generates a directly usable binary, instead of being _installed as a go package_; so I built it by directly cloning (to run from /etcd/bin). Should I still do that in `GOPATH` instead?

Reply:
Okay.
I feel it is better to keep it in GOPATH. As it is a binary, its up to you for making changes. :)

Reply:
Made changes according to your suggestions, please review